wrappers/desktop_test.go

```diff
@@ -300,8 +300,14 @@ func (s *sanitizeDesktopFileSuite) TestLangLang(c *C) {
 		// bad ones
 		{"Name[foo=bar", false},
 		{"Icon[xx]=bar", false},
+		// dbus related
+		{"Activactable=true", false},
```

Comment:
ups, I fixed that typo, the test was focused on the missing "DBus" in front, not so much on the typo :)